lnrpc/rpc.proto

```diff
@@ -708,6 +708,11 @@ message SendRequest {
     fallback.
     */
     repeated FeatureBit dest_features = 15;
+
+    /*
+    The hex-encoded payment address (32 bytes) of the generated invoice.
```

Comment:
nit: This should not mention hex encoding, since on the RPC level it's raw bytes (and when using REST, raw bytes need to be sent as base64).

Reply:
gotcha, fixed.